vnext/Microsoft.ReactNative/Views/TextInputViewManager.cpp

```diff
@@ -804,6 +805,18 @@ ShadowNode *TextInputViewManager::createShadow() const {
 
 XamlView TextInputViewManager::CreateViewCore(int64_t /*tag*/, const winrt::Microsoft::ReactNative::JSValueObject &) {
   xaml::Controls::TextBox textBox;
+  // This works around a XAML Islands bug where the XamlRoot of the first
+  // window the flyout is shown on takes ownership of the flyout and attempts
+  // to show the flyout on other windows cause the first window to get focus.
+  // https://github.com/microsoft/microsoft-ui-xaml/issues/5341
+  if (IsXamlIsland() &&
+      winrt::Windows::Foundation::Metadata::ApiInformation::IsApiContractPresent(
```

Comment:
The contract check may be redundant, I don't think we can ever have the case that IsXamlIsland() == true if we are less that API level 8.

Reply:
19h1 is the earliest version with xaml islands support which is Api contract 10 IIRC so this should be safe to remove